bash/setup_devcontainer.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#
+# Sets up enviroment for dev in local devcontainer.
+set -e
+
+# Skip when called in a github action workflow
+if [[ ${CI} != "true" ]]; then
```

Comment:
dev container shouldn't be running in CI, no?

Reply:
they actually can be (and maybe we'd wanna use them to run builds in CI sometimes?)

this script was copied from the template repo where the [dev container is run in CI](https://github.com/NYCPlanning/db-template-repo/blob/main/.github/workflows/tests_branch.yml#L51-L60) primarily to make sure it builds successfully. but we could potentially use dev containers to run production scripts to help unify local and CI environments

happy to take this out if it's more misleading than valuable

Reply:
Nah that's fine!